daniel-aguirre-salamanca/tarea-4/js/handlers.js

```diff
@@ -0,0 +1,35 @@
+import * as helpers from './helpers.js';
+import getResult from './calculator.js';
```

Comment:
@daguirreglobant , you should keep the function name "calculate" for consistency